source/common/conn_pool/conn_pool_base.cc

```diff
@@ -161,16 +161,16 @@ ConnectionPool::Cancellable* ConnPoolImplBase::newStream(AttachContext& context)
   if (!ready_clients_.empty()) {
     ActiveClient& client = *ready_clients_.front();
     ENVOY_CONN_LOG(debug, "using existing connection", client);
-    attachRequestToClient(client, context);
+    attachStreamToClient(client, context);
     // Even if there's a ready client, we may want to prefetch a new connection
     // to handle the next incoming stream.
     tryCreateNewConnections();
     return nullptr;
   }
 
   if (host_->cluster().resourceManager(priority_).pendingRequests().canCreate()) {
-    ConnectionPool::Cancellable* pending = newPendingRequest(context);
-    // This must come after newPendingRequest() because this function uses the
+    ConnectionPool::Cancellable* pending = newPendingStream(context);
```

Comment:
I can't comment on the line above this, but will the resource manager's requests also be renamed?